Misc/NEWS.d/next/Tests/2020-04-06-18-35-32.bpo-40209.XIQdFi.rst

```diff
@@ -0,0 +1 @@
+read_pyfile function in Lib/test/test_unparse.py refactor with open function in tokenize module
```

Comment:
```suggestion
Refactor ``Lib/test/test_unparse.py``'s ``read_pyfile`` with
:func:`tokenize.open`. Patch by Hakan Celik.
```

Reply:
This is not a user visible change, so it does not need a NEWS entry

Reply:
We have a tests section, I thought this was covered inside of it. For example, like this https://github.com/python/cpython/blob/master/Misc/NEWS.d/next/Tests/2020-04-02-02-14-37.bpo-40146.J-Yo9G.rst